MovieApp/MovieApp/DataProviderService.h

```diff
@@ -0,0 +1,19 @@
+#import <Foundation/Foundation.h>
+#import "ItemsArrayReceiver.h"
+#import <UIKit/UIKit.h>
+
+typedef enum Criteria
+{
+    MOST_POPULAR=0,
+    LATEST=1,
+    TOP_RATED=2
+    
+}Criterion;
+
+@interface DataProviderService : NSObject
+extern NSArray *niz;
+-(void)configure;
+-(void)getdMoviesByCriterion:(Criterion)criterion returnToHandler:(id<ItemsArrayReceiver>)delegate;
```

Comment:
`configure`  and `getdMoviesByCriterion:returnToHandler:` should be class methods (or DataProviderService should be used as singleton, in order not to repeat configuration data like base url path)